docs/components/Header/NavBar/styles.js

```diff
@@ -18,23 +18,26 @@ export const NavBar = styled.ul`
 `
 
 const activeItem = css`
+  opacity: 1;
   color: dark.900;
 
   @media (min-width: md) {
-    color: light.900;
+    opacity: 1;
+    color: white;
```

Comment:
So now for light colors we use white + opacity value ?

Reply:
yes, white stay white on dark mode, same for the header is in black and stay black in dark mode